contributors/devel/sig-node/e2e-node-tests.md

```diff
@@ -244,13 +257,15 @@ make test_e2e_node TEST_ARGS="--cgroups-per-qos=true"
 # Notes on tests run by the Kubernetes project during pre-, post- submit.
 
 The node e2e tests are run by the PR builder for each Pull Request and the results published at
```

Comment:
Whats the PR builder anyway? :eyes:

Reply:
It's prow.  The tests are run by prow.  I don't know when this sentence dates from.  Might be leftover terminology from when jenkins was used, and there was a dedicated instance for PR's